sdk/storage/azblob/appendblob/client.go

```diff
@@ -200,6 +200,12 @@ func (ab *Client) SetTier(ctx context.Context, tier blob.AccessTier, o *blob.Set
 	return ab.BlobClient().SetTier(ctx, tier, o)
 }
 
+// SetExpiry operation sets an expiry time on an existing blob. This operation is only allowed on Hierarchical Namespace enabled accounts.
+// For more information, see https://learn.microsoft.com/en-us/rest/api/storageservices/set-blob-expiry
+func (ab *Client) SetExpiry(ctx context.Context, expiryOptions blob.ExpiryOptions, o *blob.SetExpiryOptions) (blob.SetExpiryResponse, error) {
```

Comment:
The `blob.ExpiryOptions` type isn't following the design guidelines. As a result, this API sig looks weird (i.e. why two options types). Can you please rename `blob.ExpiryOptions` to `blob.ExpiryType` and the const values to match?

Reply:
Be sure to note the breaking change in the changelog.

Reply:
Done